packages/markdownlint-rule-mui/straight-quotes.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
       // closing double quote: \xE2\x80\x9D
       if (line.match('[‘’“”]')) {
         onError({
-          lineNumber,
+          lineNumber: lineNumber + 1,
```

Comment:
This was wrong, see how the lint fail with the wrong error message in https://app.circleci.com/pipelines/github/mui/material-ui/82682/workflows/e4325483-1c91-4fd7-9074-a018208f590b/jobs/437682.